website/layouts/devices/single.json.json

```diff
@@ -0,0 +1 @@
+{{ .Params | jsonify }}
```

Comment:
Check file extension

Reply:
File extension is correct check https://gohugo.io/templates/output-formats/#link-to-output-formats and search `.json.json`. I included this file as I was going to use it as an api for the react elements, in the end I found another solution and I ended up just using this for debugging. I'll remove it unless you think its useful to keep it, http://localhost:1313/device-repository/devices/netvox/r311g/index.json